dashboard/src/components/DeploymentForm/DeploymentFormBody/AdvancedDeploymentForm/AdvancedDeploymentForm.test.tsx

```diff
@@ -41,6 +41,7 @@ beforeEach(() => {
     configurable: true,
     value: jest.fn().mockImplementation(() => ({
       clearRect: jest.fn(),
+      fillRect: jest.fn(),
```

Comment:
Somehow, after merging the latest `main`, I noticed the dashboard tests were failing. It seems Monaco editor now requires `fillRect` in addition to the already mocked `clearRect` (this is from the Canvas API, which is unavailable in `jsdom`)